src/System.Slices/System/Buffers/OwnedMemory.cs

```diff
@@ -161,29 +156,20 @@ internal unsafe bool TryGetPointerInternal(long id, out void* pointer)
             return TryGetPointerCore(out pointer);
         }
 
-        unsafe bool IMemory<T>.TryGetPointer(long id, out void* pointer)
-        {
-            return TryGetPointerInternal(id, out pointer);
-        }
-
         internal bool TryGetArrayInternal(long id, out ArraySegment<T> buffer)
         {
             VerifyId(id);
             return TryGetArrayCore(out buffer);
         }
 
-        bool IMemory<T>.TryGetArray(long id, out ArraySegment<T> buffer)
-        {
-            return TryGetArrayInternal(id, out buffer);
-        }
-
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         internal Span<T> GetSpanInternal(long id, int index, int length)
         {
             VerifyId(id);
             var array = Array;
-            if (array != null)
+            if (array != null) {
                 return new Span<T>(array, Offset + index, length);
```

Comment:
I thought we put brackets on newline for methods and types and on the same line for control blocks. If not, i will change it.